web/src/main/webapp/WEB-INF/jsp/extract/createDatasetStep2.jsp

```diff
@@ -238,14 +238,15 @@ function notSelectAll() {
    --%>
 
    <td class="table_cell">
+       <fmt:message key="ungrouped" bundle="${resword}" var="ungrouped"/>
       <c:forEach var="meta" items="${item.itemMetas}" varStatus="status">
         <c:choose>
           <c:when test="${meta.groupLabel==null || meta.groupLabel==''}"></c:when>
           <c:when test="${status.last}">
-           <c:out value="${meta.groupLabel}" default="Ungrouped"/>
+           <c:out value="${meta.groupLabel}" default="${ungrouped}"/>
           </c:when>
           <c:otherwise>
-            <c:out value="${meta.groupLabel}" default="Ungrouped"/>,<br>
+            <c:out value="${meta.groupLabel}" default="${ungrouped}"/>,<br>
           </c:otherwise>
         </c:choose>
       </c:forEach>
```

Comment:
On lines 318 and 321 there are additional hardcoded English words that should be replaced with property file entries if we're going to fix i18n in this JSP file.